authorizer/rules/domainRules.js

```diff
@@ -0,0 +1,25 @@
+var SbError = require('./../../lib/SbError.js');
+module.exports = function(core, config) {
+	return function(action) {
+		var origin = action.origin,
+			room = action.room,
+			guides;
+		var error = (new SbError("AUTH:DOMAIN_MISMATCH", {
+			source: 'authorizer',
+			action: 'text'
+		}));
+		if(action.type === "room") room = action.old;
+		if (!origin || !room) return error;
+		guides = room.guides;
+		if (origin.verified !== true) {
+			return error;
```

Comment:
If the origin is not verified, we dont allow any action. This is just a taking a safer approach.